### DISCUSSION TITLE
Welcome to Dantherm and Pluggit Integration Discussions!

### DISCUSSION THREAD

**Discussion by Tvalley71** [Tvalley71/dantherm#36] [Announcements]
## 👋 Welcome to the Dantherm and Pluggit Integration GitHub Discussion!

Hello everyone,

Welcome to the GitHub discussion for the Dantherm and Pluggit Integration for Home Assistant! I'm excited to have you here as we continue to enhance and refine this integration.

This space is dedicated to addressing issues, answering questions, and exploring new ideas for improvement. Your input and collaboration are wecome in making this project even better. Here are a few guidelines to keep our discussions productive:

+ **Report Issues:** If you encounter any problems, please open an issue with detailed information to help diagnose and resolve it.
+ **Ask Questions:** Feel free to ask any questions you have about using the integration. Your inquiries help improve documentation and user experience.
+ **Share Ideas:** Have suggestions for new features or improvements? Share your ideas here so we can discuss and potentially implement them.
+ **Translation Contributions:** If you can provide translations for different languages, your help would be greatly appreciated to make this integration more accessible globally.

Looking forward to our collaborative efforts to make the integration more effective and user-friendly.

Thank you for your participation and support!

Best regards,

Tvalley71
Solo Developer
Dantherm Integration for Home Assistant
Pluggit Integration for Home Assistant

**Comment by Ramseswisse** [comment]
Bonjour,

Je vous remercie déjà pour votre travail et pour l'intégration Dantherm que vous avez créé. Avant j'avais un Sensor modbus en tcp dans le yaml qui remontait les infos bizarrement, récupérer sur hacf ou les interrogations se faisaient par récupération des codes bus qui n'étaient pas les bonnes pour moi. Je vous confirme que vous pouvez ajouter que votre travail est compatible avec la ventilation Dantherm HCV 700.

Merci.

**Reply by Tvalley71** [reply]
Bonjour,

Merci beaucoup pour votre retour et votre confirmation. Je suis ravi d'apprendre que l'intégration que j'ai créée fonctionne correctement avec votre système de ventilation Dantherm HCV 700. Je vais ajouter cette information afin de préciser la compatibilité.

N’hésitez pas à revenir vers moi si vous avez d'autres retours ou questions (ChatGPT).

Tvalley71
